setup.cfg

```diff
@@ -22,13 +21,34 @@ keywords =
 packages =
     spotify_tensorflow
 
-[options.extras_require]
-tensorflow = tensorflow==1.9.0
-tensorflow_gpu = tensorflow-gpu==1.9.0
+[extras]
+tensorflow = 
+    tensorflow==1.9.0
+tensorflow_gpu = 
+    tensorflow-gpu==1.9.0
+tfx =
+    luigi>=2.3.3,<3.0.0
+    google-api-python-client>=1.5.4,<2.0
+    oauth2client<4.0.0
+    tensorflow-data-validation==0.11.0
```

Comment:
ditto: pinning

Reply:
See comment above.

Also, this is not a change from master, where tfdv is pinned in `test-requirements.txt`